ui/raidboss/data/03-hw/trial/sephirot.ts

```diff
@@ -0,0 +1,93 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.ContainmentBayS1T7,
+  triggers: [
+    {
+      id: 'Sephirot Fiendish Rage',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0048' }),
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Sephirot Ratzon',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0046' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Sephirot Ain',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '16DD', source: 'Sephirot', capture: false }),
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Sephirot Earth Shaker',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0028' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.earthshaker(),
+    },
+    {
+      id: 'Sephirot Pillar of Mercy',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '16EA', source: 'Sephirot' }),
+      alertText: (_data, matches, output) => {
+        // The pillars spawn in slightly randomized positions, typically
+        // either in the west side, center, or east side. The X position
+        // range for the arena is -20 to 20.
+        //
+        // TODO: Sometimes the coordinates for the Pillar of Mercy seem to be
+        // wrong (an xPos of -10 for a central knockback or +12.5 for a west
+        // knockback). It's unclear if this is a bug in the parser or
+        // cactbot.
```

Comment:
For mobs that spawn and immediately start casting, sometimes they are moved into place right before they cast, and the position is wrong because it hasn't been updated in the plugin memory yet.

If this is the "blue knockback puck", I think you could just say "knockback" on its own (or nothing).

Reply:
The casts come from Sephirot, (but he is in big set piece mode), so I am not sure that's it. But I'll stick with just knockback and simplify it to avoid this.